Source/Core/DiscIO/ScrubbedBlob.cpp

```diff
@@ -37,6 +37,12 @@ std::unique_ptr<ScrubbedBlob> ScrubbedBlob::Create(const std::string& path)
   return std::unique_ptr<ScrubbedBlob>(new ScrubbedBlob(std::move(blob), std::move(scrubber)));
 }
 
+std::unique_ptr<BlobReader> ScrubbedBlob::CopyReader() const
+{
+  return std::unique_ptr<ScrubbedBlob>(
+      new ScrubbedBlob(m_blob_reader->CopyReader(), std::move(m_scrubber)));
```

Comment:
Using std::move on a member variable is a correctness issue. The old object needs to remain valid.